hledger-lib/Hledger/Reports/AccountTransactionsReport.hs

```diff
@@ -64,26 +65,20 @@ import Hledger.Utils
 -- posts to the current account), most recent first.
 -- Reporting intervals are currently ignored.
 --
-type AccountTransactionsReport =
-  (String                          -- label for the balance column, eg "balance" or "total"
-  ,[AccountTransactionsReportItem] -- line items, one per transaction
-  )
+type AccountTransactionsReport = [AccountTransactionsReportItem] -- line items, one per transaction
```

Comment:
The idea of including titles in Reports was to make it easy for client-side (JS) UIs to consume and render them. In some cases it might not be easy for them to generate meaningful titles. However there's the issue of i18n, and removing them does simplify things, so I agree with this general removal of titles from Reports I think.